examples/map.birb

```diff
@@ -2,4 +2,11 @@ Map map = {
     "content": "This is a message"
 };
 
-scremLn(map["content"]); // output This is a message
\ No newline at end of file
+scremLn(map["content"]); // output This is a message
+scremLn(map.isEmpty); // output false
+scremLn(map.isNotEmpty); // output true
+scremLn(map.length); // output 1
+scremLn(map.hashCode); // ignore
```

Comment:
prefer validating what is known about the value instead of ignoring. for example in this case, we know it should be an int, so outputting the runtime type of it and expecting it to be an int is better here.